### PR TITLE
update slurm.conf

### PR DESCRIPTION
**Pull request by andriy-safe-ai**
Add slurm config from prod to source. 

Deployed a cpu dev cluster with the updated template.

### CODE REVIEW

File: playbooks/roles/slurm/templates/slurm.conf.j2 [hunk 1]
```diff
@@ -35,20 +36,49 @@ SelectTypeParameters=CR_Core
 AccountingStorageHost={{ bastion[0] }}
 AccountingStorageType=accounting_storage/slurmdbd
 TaskPlugin=task/affinity,task/cgroup
+AccountingStorageTRES=gres/gpu,gres/gpu:A100
```
Comment (steven-basart) [inline playbooks/roles/slurm/templates/slurm.conf.j2:39]: This one is at the bottom of this file please edit it there or double check.

**Review by steven-basart — CHANGES_REQUESTED** [review]
Double check that the keys don't already exist such as the ones I pointed out. 

Also link to the Github issues. I think I have it as one or more issues that this PR is trying to address.

File: playbooks/roles/slurm/templates/slurm.conf.j2 (hunk 1 above)
Comment (andriy-safe-ai) [reply playbooks/roles/slurm/templates/slurm.conf.j2:39]: I just checked `slurm.conf` in prod and this is where it is located.

File: playbooks/roles/slurm/templates/slurm.conf.j2 [hunk 2]
```diff
@@ -35,20 +36,49 @@ SelectTypeParameters=CR_Core
...
+
+# This next group determines the weighting of each of the
+# components of the Multi-factor Job Priority Plugin.
+# The default value for each of the following is 1.
+PriorityWeightAge=1000
+PriorityWeightFairshare=10000
+PriorityWeightJobSize=1000
 
 {% if sacct_limits|bool %}
 AccountingStorageTRES=gres/gpu
```
Comment (steven-basart) [inline playbooks/roles/slurm/templates/slurm.conf.j2:84]: here

File: playbooks/roles/slurm/templates/slurm.conf.j2 (hunk 1 above)
Comment (steven-basart) [reply playbooks/roles/slurm/templates/slurm.conf.j2:39]: I'm saying this file has these variables already defined below.  Please see [comment](https://github.com/centerforaisafety/cerberus-cluster/pull/134/files#r1242593982) marked

**Review by steven-basart — CHANGES_REQUESTED** [review]
Please fix

File: playbooks/roles/slurm/templates/slurm.conf.j2 (hunk 1 above)
Comment (steven-basart) [reply playbooks/roles/slurm/templates/slurm.conf.j2:39]: If you wish to move them here to be consistent with prod that's fine but we can't doubly define the variables.

Comment (andriy-safe-ai) [reply playbooks/roles/slurm/templates/slurm.conf.j2:39]: @steven-basart 
Ok. I see it now.

File: playbooks/roles/slurm/templates/slurm.conf.j2 (hunk 2 above)
Comment (andriy-safe-ai) [reply playbooks/roles/slurm/templates/slurm.conf.j2:84]: Thanks

**Review by steven-basart — APPROVED** [review]
LGTM

**Pull request merged**